Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Events.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.Foundation;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The <see cref="ImageCropper"/> control allows user to crop image freely.
+    /// </summary>
+    public partial class ImageCropper
+    {
+        private void ImageCropperThumb_KeyDown(object sender, KeyRoutedEventArgs e)
```

Comment:
Discussion point: Do we really think users will use the keyboard to tab to a thumb and change the size of an item?

Reply:
Users may not use it most of the time, but this ensures that it can be manipulated when the mouse or touch screen is unavailable.

Reply:
Keyboard accessibility thoughts are always good.